### PR TITLE
rename package name of native client

### PR DESCRIPTION
**Pull request by critical27**

### CODE REVIEW

**Comment by critical27** [comment]
Jenkins go

File: src/tools/native-client/pom.xml [hunk 1]
```diff
@@ -72,14 +72,12 @@
                             <goal>javah</goal>
                         </goals>
                         <configuration>
-                            <javahOutputDirectory>${basedir}/src/main/cpp
-                            </javahOutputDirectory>
+                            <javahOutputDirectory>${basedir}/src/main/cpp</javahOutputDirectory>
                             <javahOutputFileName>
-                                com_vesoft_client_NativeClient.h
+                                com_vesoft_nebula_codec.h
```
Comment (darionyaphet) [inline src/tools/native-client/pom.xml]: the javah output file name should be `package name_file name` ?

Comment (critical27) [reply src/tools/native-client/pom.xml]: Ok

File: src/tools/native-client/pom.xml [hunk 2]
```diff
@@ -151,6 +151,61 @@
                     <argLine>-Djava.library.path=_build/</argLine>
                 </configuration>
             </plugin>
+            <plugin>
```
Comment (darionyaphet) [inline src/tools/native-client/pom.xml]: would you publish natice codec to maven center ?

File: src/tools/native-client/pom.xml [hunk 3]
```diff
@@ -151,6 +151,61 @@
...
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-javadoc-plugin</artifactId>
+                <version>3.1.1</version>
+                <configuration>
+                    <excludePackageNames>com.facebook.thrift:com.facebook.thrift.*</excludePackageNames>
```
Comment (darionyaphet) [inline src/tools/native-client/pom.xml]: should not exclude thrift

File: src/tools/native-client/pom.xml (hunk 2 above)
Comment (critical27) [reply src/tools/native-client/pom.xml]: Yeah, then we can use in nebula-java

File: src/tools/native-client/pom.xml (hunk 3 above)
Comment (critical27) [reply src/tools/native-client/pom.xml]: Make sense.

File: src/tools/native-client/src/main/java/com/vesoft/client/NativeClientResourceLoader.java [hunk 4]
```diff
@@ -1,47 +0,0 @@
-/* Copyright (c) 2018 vesoft inc. All rights reserved.
- *
```
Comment (darionyaphet) [inline src/tools/native-client/src/main/java/com/vesoft/client/NativeClientResourceLoader.java]: why remove `NativeClientResourceLoader`, maybe use `NativeClientResourceLoader.resourceLoader();` could load `.so`

Comment (critical27) [reply src/tools/native-client/src/main/java/com/vesoft/client/NativeClientResourceLoader.java]: We need to use it in nebula-java, but not here? We just need to pack the `so` in jar in codec.

**Pull request closed without merging**